packages/workshop-app/app/routes/_app+/$exerciseNumber.tsx

```diff
@@ -102,9 +102,11 @@ export default function ExerciseNumberRoute() {
 							{data.exercise.title}
 						</h1>
 						<div className="mt-8">
-							<ButtonLink to="01/problem" prefetch="intent" varient="big">
-								Start Learning
-							</ButtonLink>
+							<div className="clip-path-button-outer bg-black">
```

Comment:
instead of having to remember to wrap it each time, it would make more sense to add it to Button component here: https://github.com/epicweb-dev/kcdshop/blob/main/packages/workshop-app/app/components/button.tsx#L70

Reply:
I will try it... 
the button style need more work...